modules/admin/client/components/AdminThreads.component.js

```diff
@@ -0,0 +1,149 @@
+// External dependencies
+import classnames from 'classnames';
+import React, { useState } from 'react';
+
+// Internal dependencies
+import { getThreads } from '../api/threads.api';
+import AdminHeader from './AdminHeader.component';
+import Json from './Json.component';
+import UserLink from './UserLink.component';
+import TimeAgo from '@/modules/core/client/components/TimeAgo';
+
+// Mongo ObjectId is always 24 chars long
+const MONGO_OBJECT_ID_LENGTH = 24;
+
+export default function AdminThreads() {
+  const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Same as above. It may be nice to have a TODO comment about replacing this with `useLocation` of react-router if/when we use it.

Reply:
Aside, we could actually split the admin app to be a little more separate on clientside and start experimenting with react-router for it already now.